src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -227,10 +227,11 @@ public Index checkIfRoomPresent(Integer roomNumber) {
 
     @Override
     public void updateRoomListWhenPatientsChanges(Patient patientToEdit, Patient editedPatient) {
```

Comment:
I personally don't really like this method because it's kind of weird to call this method in the command classes. But I'm not sure how else to go about it either :/

Reply:
ya i think this one we dont have a choice but to couple them